examples/example4/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2012 The GoSNMP Authors. All rights reserved.  Use of this
+// source code is governed by a BSD-style license that can be found in the
+// LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	g "github.com/gosnmp/gosnmp"
```

Comment:
Sorry, one last nitpick. 
Preferably have the remote modules listed last in the import as such:
```go
import (
  "fmt"
  "log"
  "os"
  "time"
  
  g "github.com/gosnmp/gosnmp"
)

```